redis/src/storages/redis/impl/secdist_redis.hpp

```diff
@@ -21,6 +21,7 @@ struct RedisSettings {
     std::vector<std::string> shards;
     std::vector<HostPort> sentinels;
     storages::redis::Password password{std::string()};
+    storages::redis::Password sentinel_password{std::string()};
```

Comment:
why not just reuse `password`? Looks like `password` and `sentinel_password` are newer used at the same time

Reply:
It depends on security rules. In some yellow bank it have to be different and authentication is on sentinel and on redis connection too.